external_server/server_module/command_waiting_thread.py

```
@@ -0,0 +1,143 @@
+from typing import Callable, Optional
```

Ruff

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)